examples/client/tui/common.nim

```diff
@@ -178,6 +183,7 @@ const
     "create": "createaccount",
     "delete": "deletewalletaccount",
     "deletetoken": "deletecustomtoken",
+    "getprice": "getprice",
```

Comment:
No need to add to the aliases table if there's no alias for `getprice`.

Reply:
Done.